packages/web-components/src/components/ic-search-bar/ic-search-bar.tsx

```diff
@@ -914,7 +928,7 @@ export class SearchBar {
                 onBlur={this.handleSubmitSearchBlur}
                 onFocus={this.handleSubmitSearchFocus}
                 onKeyDown={this.handleSubmitSearchKeyDown}
-                type="submit"
+                type={getButtonType()}
```

Comment:
just a slight optimisation suggestion. At the mo, the `getButtonType` function will get called every time the component renders, so whenever another unrelated prop changes, it will result in a call to this function. 

As a optimisation, you could have the button type as a state variable, initialised in the `componentWillLoad` & then place a watch on the `preventFormSubmitOnSearch` for when it changes, to update the value